lib/delayed_job/delayed_worker.rb

```diff
@@ -25,6 +25,8 @@ def start_working
     setup_metrics(config) if @publish_metrics
     BackgroundJobEnvironment.new(config).setup_environment(readiness_port)
 
+    # load monkey patch for sequel backend to support configurable job lock method (postgres only)
+    require 'delayed_job/sequel_patch'
```

Comment:
`require` might fit better in `db.rb`:
https://github.com/cloudfoundry/cloud_controller_ng/blob/9cf8c4581aaab3d361e5dc694fb97b86a315dbfa/lib/cloud_controller/db.rb#L87-L87

Reply:
Moved and works well.